src/rules/noUnusedExpressionRule.ts

```diff
@@ -38,17 +38,17 @@ class NoUnusedExpressionWalker extends Lint.RuleWalker {
         this.expressionIsUnused = true;
         super.visitExpressionStatement(node);
         if (this.expressionIsUnused) {
-            // ignore valid unused expressions
-            if (node.expression.kind === ts.SyntaxKind.StringLiteral) {
-                const expressionText = node.expression.getText();
-                if (expressionText === "\"use strict\"" || expressionText === "'use strict'") {
-                    return;
-                }
-            } else if (node.expression.kind === ts.SyntaxKind.DeleteExpression || node.expression.kind === ts.SyntaxKind.YieldExpression) {
-                return;
-            }
+            const { expression } = node;
+            const { kind } = expression;
+            const isValidStandAloneExpressionKind = kind === ts.SyntaxKind.DeleteExpression
```

Comment:
nit: "standalone" is one word, not 2, and I think `Kind` is useless in this identifier

i.e. `isValidStandaloneExpression`